sdk/security/keyvault/azkeys/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.0-beta.1
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.1
-	github.com/Azure/azure-sdk-for-go/sdk/internal v1.4.0
+	github.com/Azure/azure-sdk-for-go/sdk/internal v1.3.0
```

Comment:
Wait...just noticed this is a downgrade. Is that expected?

Reply:
Having issues with the newer version of sdk/internal. Its recording package is breaking my test proxy and causing the pipelines to fail. Working fixing it now

Reply:
fixed in [f1a71c1](https://github.com/Azure/azure-sdk-for-go/pull/21910/commits/f1a71c116e1b8fd1bea0a1836a658ec274a90756)